sdk/monitor/monitor-opentelemetry-exporter/review/monitor-opentelemetry-exporter.api.md

```diff
@@ -16,8 +20,21 @@ export interface AzureExporterConfig {
     connectionString?: string;
 }
 
+// Warning: (ae-forgotten-export) The symbol "AzureMonitorBaseExporter" needs to be exported by the entry point index.d.ts
```

Comment:
`AzureMonitorBaseExporter` should be exported

Reply:
Users will only use AzureMonitorMetricExporter or AzureMonitorTraceExporter, any particular reason why this need to be exported as well?

Reply:
The customer may not need to instantiate it directly, but it is still needed by typescript for typechecking purposes. As a rule of thumb, anything part of the public surface should be exported.